### ISSUE TITLE
when dpad-minimizing is disabled - hit the mid to attack

### ISSUE THREAD

**Issue by GoogleCodeExporter** [alistairreilly/andors-trail#246]
```
Currently it works like this:
1) When dpad-minimizing is disabled:
Hit the mid results in minimizing of the dpad.
2) When dpad-minimizing is disabled:
Hit the mid results in no action

It should work like this:
1) When dpad-minimizing is disabled:
Hit the mid results in minimizing of the dpad.
2) When dpad-minimizing is disabled:
Hit the mid results in an attack (when in combat mode)

Patch attached.
```

Original issue reported on code.google.com by `SamuelPl...@gmail.com` on 5 Oct 2011 at 7:34

Attachments:
- [when dpad-minimizing is disabled - hit the mid to attack.patch](https://storage.googleapis.com/google-code-attachments/andors-trail/issue-246/comment-0/when dpad-minimizing is disabled - hit the mid to attack.patch)

**Comment by GoogleCodeExporter** [comment]
```
Thanks, looks good. Committed in r168.
```

Original comment by `oskar.wi...@gmail.com` on 7 Oct 2011 at 8:57
- Changed state: **Ready**
- Added labels: **Milestone-0.6.10**

**Comment by GoogleCodeExporter** [comment]
```
Released in v0.6.10 . Closing.
```

Original comment by `oskar.wi...@gmail.com` on 21 Oct 2011 at 8:56
- Changed state: **Fixed**
